test/fluree/db/query/history_test.clj

```diff
@@ -177,357 +177,359 @@
-          db5 @(test-utils/transact ledger {:id :ex/alice
-                                            :ex/x "foo-cat"
-                                            :ex/y "bar-cat"}
-                                    {:message "meow"})]
+          db1    @(test-utils/transact ledger {:id   :ex/alice
+                                               :ex/x "foo-1"
+                                               :ex/y "bar-1"})
+          db2    @(test-utils/transact ledger {:id   :ex/alice
+                                               :ex/x "foo-2"
+                                               :ex/y "bar-2"})
+          db3    @(test-utils/transact ledger {:id   :ex/alice
+                                               :ex/x "foo-3"
+                                               :ex/y "bar-3"})
+          db4    @(test-utils/transact ledger {:id   :ex/cat
+                                               :ex/x "foo-cat"
+                                               :ex/y "bar-cat"})
+          db5    @(test-utils/transact ledger {:id   :ex/alice
+                                               :ex/x "foo-cat"
+                                               :ex/y "bar-cat"}
+                                       {:message "meow"})]
       (testing "at time t"
         (is (= [{:f/commit
                  {:f/address
-                  "fluree:memory://2a2436a01df3343870bca46e3a24c6b57df73f28666fe5247c221ca888abba5e",
-                  :f/v 0,
-                  :f/time 720000,
+                  "fluree:memory://3f7ff6df48e007cab36098274fd822ac11c9da4bf8b29762d1de3fdbdd6b6013",
+                  :f/v      0,
+                  :f/time   720000,
                   :id
-                  "fluree:commit:sha256:bb7lpextw2b64rq2k3ilcttpb66he3derz4cvnqtu5znptvzfndev",
+                  "fluree:commit:sha256:bsso2btsgd4gsukmqrlmm4gap4gbmk5fkemiht6hlpjkaxzdgmmk",
                   :f/branch "main",
                   :f/data
                   {:f/address "fluree:memory://5d3ce686baa6fd5cc547b5e03e6aca3d92cbce0328c2320a49c514b01e58b4c2",
-                   :f/flakes 11,
-                   :f/size 996,
-                   :f/t 1,
-                   :f/assert [{:ex/x "foo-1", :ex/y "bar-1", :id :ex/alice}],
+                   :f/flakes  11,
+                   :f/size    996,
+                   :f/t       1,
+                   :f/assert  [{:ex/x "foo-1", :ex/y "bar-1", :id :ex/alice}],
                    :f/retract []},
                   "https://www.w3.org/2018/credentials#issuer"
                   {:id "did:fluree:TfCzWTrXqF16hvKGjcYiLxRoYJ1B8a6UMH6"},
-                  :f/alias "committest",
+                  :f/alias  "committest",
                   :f/context
-                  "fluree:memory:///contexts/b6dcf8968183239ecc7a664025f247de5b7859ac18cdeaace89aafc421eeddee"}}]
+                  "fluree:memory://b6dcf8968183239ecc7a664025f247de5b7859ac18cdeaace89aafc421eeddee"}}]
```

Comment:
I haven't fully investigated this, but I wonder if removing the `/contexts/` portion of this path (which was done here https://github.com/fluree/db/pull/388/commits/2c31c33fd9951f130f0561ed7a42f24b9c7405b8) affected the SHAs and sizes here. Since we store this commit metadata in flakes for the `/history` endpoint, changes like this will affect the calculations on the resulting db.

Reply:
Yeah that could be.